scalableminds/webknossos
# Panel content reorganization + Control/Tool rework

### URL of deployed dev instance (used for testing):
- https://panelcontentreorganization.webknossos.xyz

### Steps to test:
- Look at the checklist below and check whether the panels/tabs have the content as described in the list.
- Please also check the (basic) functionality of the panel elements / settings.

- I had quite a lot of merging conflicts relating the dark mode pr / css. This needs to be tested, so nothing breaks because of this PR.

### Issues:
- fixes #5231

------
